.github/workflows/mayhem-api.yml

```diff
@@ -0,0 +1,97 @@
+name: "Mayhem API analysis"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  mayhem:
+    name: Mayhem API analysis
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    env:
+      DB_DATABASE: domjudge
+      DB_USER: user
+      DB_PASSWORD: password
+    steps:
+      - uses: actions/checkout@v2
+
+      # Install default DOMjudge dommserver apt packages
+      - name: Install domserver required apt packages
+        run: |
+          sudo apt update
+          sudo apt install -y acl zip unzip nginx php php-fpm php-gd \
+                              php-cli php-intl php-mbstring php-mysql php-curl php-json \
+                              php-xml php-zip composer ntp make sudo debootstrap \
+                              libcgroup-dev lsof php-cli php-curl php-json php-xml \
+                              php-zip procps gcc g++ default-jre-headless \
+                              default-jdk-headless ghc fp-compiler autoconf automake bats \
+                              python3-sphinx python3-sphinx-rtd-theme rst2pdf fontconfig \
+                              python3-yaml latexmk
+
+      - name: Install composer files
+        run: |
+          composer install --no-scripts
+          composer run-script package-versions-dump
+
+      # Patch the operationId to be unique again to conform to OpenApi standard
+      - name: Patch upstream
+        run: sudo sed -i "s/\$operation->operationId =/#\$operation->operationId =/g" lib/vendor/nelmio/api-doc-bundle/OpenApiPhp/DefaultOperationId.php
+
+      # We install in the current directory to keep mounts possible etc.
+      - name: Configure Makefile
+        run: |
+          DIR=$(pwd)
+          make configure
+          ./configure --with-baseurl='http://localhost/domjudge/' --enable-doc-build=no --prefix=$HOME/domjudge
+
+      # sudo because its default in the manual
+      - name: Compile domserver
+        run: |
+          make domserver
+          sudo make install-domserver
+
+      # GitHub action sometimes doesnt start the SQL server.
```

Comment:
https://github.blog/changelog/2020-02-21-github-actions-breaking-change-ubuntu-virtual-environments-will-no-longer-start-the-mysql-service-automatically/

I think 'sometimes' is 'always'

Reply:
I'll remove the comment